src/lib/Scenes/Show/Screens/MoreInfo.tsx

```diff
@@ -47,6 +47,22 @@ export class MoreInfo extends React.Component<Props, State> {
       })
     })
 
+    if (show.openingReceptionText) {
+      if (show.press_release) {
```

Comment:
Is the nesting of these `if`s intentional? I'm not seeing how these are dependent on one another.

Reply:
Good call this was an error.